docs/index.html

```diff
@@ -111,8 +111,8 @@ <h3 class="center">
         </h3>
         <div class="contact">
             <a class="button primary"
-               href="mailto:betatester@platzhalter.io?subject=Anmeldung%20zum%20Testen&body=Hallo%20zusammen%2C%0A%0Aich%20m%C3%B6chte%20mich%20hiermit%20als%20Beta-Tester%20f%C3%BCr%20Platzhalter.io%20anbieten!%0A%0A%23Platz%20f%C3%BCr%20dein%20Feedback%20zur%20Idee%23%0A%0ABitte%20nehmt%20mich%20in%20Eure%20Verteiler%20mit%20der%20Mail-Adresse%20%23deine%20E-Mail-Adresse%23%20auf.%0A%0AIch%20bin%20damit%20einverstanden%2C%20dass%20Ihr%20meine%20E-Mail-Adresse%20zur%20Kontaktaufnahme%20bez%C3%BCglich%20des%20Tests%20und%20zugeh%C3%B6riger%20Umfragen%20speichert.%20Mir%20ist%20bewusst%2C%20dass%20ihr%20Dienste%20wie%20Google%20Forms%20oder%20%C3%A4hnliches%20einsetzen%20werdet%2C%20um%20z%C3%BCgig%20voran%20zu%20kommen%20und%20mir%20ist%20klar%2C%20dass%20ihr%20f%C3%BCr%20entsprechende%20Einladungen%20auch%20dort%20die%20genannte%20E-Mailadresse%20verwenden%20werdet.%0A%0AViele%20Gr%C3%BC%C3%9Fe%0A%23dein%20Name%20und%20Unternehmen%23">
-                Zum Beta-Test anmelden!
+               href="https://beta.platzhalter.io/">
```

Comment:
Do you think it makes sense to hand out the link directlly? We got a lot of valuable feedback from the users who went through it with us in the webinar format

Reply:
Yes, but we didn't get any more replies for our webinars and we're ready to go out there